sigstore/_internal/fulcio/client.py

```diff
@@ -212,16 +210,19 @@ def post(
             except (AttributeError, KeyError):
                 raise FulcioClientError from http_error
 
+        if resp.json().get("signedCertificateEmbeddedSct"):
+            sct_embedded = True
+            certificates = resp.json()["signedCertificateEmbeddedSct"]["chain"]["certificates"]
+        else:
+            sct_embedded = False
+            certificates = resp.json()["signedCertificateDetachedSct"]["chain"]["certificates"]
+
         # Cryptography doesn't have chain verification/building built in
         # https://github.com/pyca/cryptography/issues/2381
-        try:
-            cert_pem, *chain_pems = pem.parse(resp.content)
-            cert = load_pem_x509_certificate(cert_pem.as_bytes())
-            chain = [load_pem_x509_certificate(c.as_bytes()) for c in chain_pems]
-        except ValueError:
-            raise FulcioClientError(f"Did not find a cert in Fulcio response: {resp}")
+        cert = load_pem_x509_certificate(certificates[0].encode())
```

Comment:
Would be good to check for length here or at least catch `IndexError`.

Reply:
Added a check for length.